sos/report/plugins/ssh.py

```diff
@@ -84,6 +84,6 @@ def user_ssh_files_permissions(self):
         for user in users_data:
             home_dir = self.path_join(user.pw_dir, '.ssh')
             if self.path_isdir(home_dir):
```

Comment:
is it worth removing this check now, as you're checking if the path exists in `add_dir_listing()`